client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -14,23 +16,40 @@ const { Search } = Input;
  */
 const RobotOverview = () => {
   const [searchValue, setSearchValue] = useState('');
-  const [userId, setUserId] = useState(1);
+  const [userId, setUserId] = useState('80625d115100a2ee8d8e695b');
+  const [robotList, setRobotList] = useState([]);
+
+  /**
+   * @description Fetches Bots for the specified user and will trigger a rerender so that it will be displayed
+   * @param {String} userIuserIdToFetch The userId to fetch Bots for
+   */
+   const retrieveBotList = async (userIuserIdToFetch) => {
```

Comment:
Shouldn't be async as we handle fetchSSOTsForUser with .then and not with await.